src/NativeScript/CMakeLists.txt

```diff
@@ -195,6 +195,12 @@ set_target_properties(NativeScript PROPERTIES
     "
 )
 
+add_custom_command(TARGET NativeScript
```

Comment:
Yeah, that is much better. I was missing the `SetActiveArchitectures` macro for that one to work :smile: